emgapi/views.py

```diff
@@ -300,7 +300,7 @@ def top10(self, request):
             SAMPLE as sample
         WHERE node.lft BETWEEN parent.lft AND parent.rgt
             AND node.BIOME_ID = sample.BIOME_ID
-            AND sample.IS_PUBLIC = 1
+            AND NOT sample.IS_PRIVATE
```

Comment:
I wonder whether this is even needed? I'd be surprised if a certain biome is oversampled by private studies to such an extent that it changes the top10 order.

Reply:
I agree, I'm going to add a comment there. We can fix that later.